test/gas-estimation/InterchainGasEstimation.js

```diff
@@ -36,4 +41,18 @@ describe('InterchainGasEstimation', () => {
 
         expect(estimate).to.equal(353400090264);
     });
+
+    it('should compute gas estimate correctly', async () => {
```

Comment:
add any missing coverage as well?

Reply:
Added full coverage, 1 revert line is unreachable.